geode-cq/src/test/java/org/apache/geode/internal/cache/PutAllCSDUnitTest.java

```diff
@@ -500,6 +502,81 @@ public void run2() throws CacheException {
   }
 
   /**
+   * Create PR without redundancy on 2 servers with lucene index. Feed some key s. From a client, do
+   * removeAll on keys in server1. During the removeAll, restart server1 and trigger the removeAll
+   * to retry. The retried removeAll should return the version tag of tombstones. Do removeAll again
+   * on the same key, it should get the version tag again.
+   */
+  @Test
+  public void test51871() throws CacheException, InterruptedException {
```

Comment:
Will it be possible to change the test name to follow the <expected test result>when<condition tested>() naming convention